src/user/User.js

```diff
@@ -120,11 +118,14 @@ module.exports = {
   async _loadInitialUser() {
     var value = null;
     try {
-      value = await AsyncStorage.getItem(TOKEN_KEY);
+      value = await Data._options.AsyncStorage.getItem(TOKEN_KEY);
     } catch (error) {
       console.warn('AsyncStorage error: ' + error.message);
     } finally {
       this._loginWithToken(value);
     }
   },
+  setItem(){
+
+  }
```

Comment:
Hi! Quick question: has this method been added purposefully or is it old code?

Reply:
Missed that one.  Dead code, and I'll push a new update with it removed.  I was planning on abstracting the methods: `setItem`, `getItem`, and `removeItem` to allow even wider storage solutions, but thought better of it..